lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -69,17 +67,23 @@ loadbalance_algorithm:
   get_command: "show run fabricpath all"
   get_value: '/^fabricpath load\-balance (\S+)\s*$/'
   set_value: "<state> fabricpath load-balance <algo>"
-  N5k:
+  N5k: &loadbal_alg_n5_n6
```

Comment:
Agreed, but if we add n8k, and n9k then we have to modify `loadbal_lg_n5_n6` to become `loadbal_lg_n5_n6_n8_n9`.  Not that big a deal so not blocking.